policyengine_us/tests/policy/baseline/gov/states/ca/tax/income/ca_income_tax.yaml

```diff
@@ -0,0 +1,20 @@
+- name: High-income taxpayer
```

Comment:
We now only include unit tests for variables with `formula`s, so we can remove this.

Reply:
OK.  I'll remove the unit tests.